### ISSUE TITLE
Image support

### ISSUE THREAD

**Issue by vasiliybondarenko** [norbert-radyk/spoiwo#48]
The library lacks image support, in particular inserting image in header.

**Comment by norbert-radyk** [comment]
Please provide a sample of the code supporting the missing functionality via Apache POI, but which is not supported in spoiwo. I will try to add it once given clarity about the requirement.

**Comment by vasiliybondarenko** [comment]
@norbert-radyk 
```scala
  def createPicture(wb: XSSFWorkbook, imageFileName: String) = {
	val sheet = wb.getSheetAt(0)

	val is = new FileInputStream(imageFileName)
	val bytes = IOUtils.toByteArray(is)
	val pictureIdx = wb.addPicture(bytes, Workbook.PICTURE_TYPE_JPEG)
	val helper = wb.getCreationHelper
	val drawing = sheet.createDrawingPatriarch

	val anchor = helper.createClientAnchor
	anchor.setAnchorType(AnchorType.MOVE_DONT_RESIZE)

	anchor.setCol1(0)
	anchor.setCol2(0)

	val pict = drawing.createPicture(anchor, pictureIdx)
	pict.resize()
	wb
  }
```

**Comment by vasiliybondarenko** [comment]
Can I create a PR?

**Comment by norbert-radyk** [comment]
https://github.com/norbert-radyk/spoiwo/pull/50 was merged and will propagate as v1.6.0 in the next 24h.
